doc/HOWTO_oracle.md

```diff
@@ -13,15 +13,22 @@ Follow these instructions to set up the Oracle database to run in Docker, but th
 * Start the oracle container:
 
 ```
-docker run -d -p 1521:1521 --name nucore_db sath89/oracle-12c
-# wait, it sometimes takes a few minutes to come up
-# "ORA-01033: ORACLE initialization or shutdown in progress" means wait.
+mkdir $HOME/oracle_data
+docker run -p 1521:1521 --name nucore_db -v $HOME/oracle_data:/u01/app/oracle sath89/oracle-12c
+# This will take several minutes as the database initializes.
+# Wait for "Database ready to use. Enjoy! ;)"
 ```
 
+`$HOME/oracle_data` is just a recommended location where your oracle data files will
+be saved. This will allow them to remain persistent even across re-creations of
+the container.
+
 * Next time you want to start the server, run:
 
 ```
 docker start nucore_db
+# wait, it sometimes takes a few minutes to come up
```

Comment:
Maybe make this line `sleep 60 # # wait, it sometimes takes a while for the container to start` for copy & pasteability?